app/src/main/java/ch/admin/bag/dp3t/util/NotificationUtil.java

```diff
@@ -33,13 +33,36 @@
 	public static final int NOTIFICATION_ID_UPDATE = 43;
 	public static final int NOTIFICATION_ID_REMINDER = 44;
 
+	public static void generateContactNotification(Context context) {
```

Comment:
There is some duplicated code now in the NotificationUtil class that could be cleaned up.